### ISSUE TITLE
Incorrect NFTool Behavior

### ISSUE THREAD

**Issue by MiLoserDie** [ketoo/NoahGameFrame#169]
I spent about 2 hours to turn my Excel file into the correct .ini file, but I did not succeed.
Can you make the correct NFTool so that it can properly parse Excel files? I do not know what it is connected with, but many Excel files that I create when generating in .ini turn into the wrong formats!

Need fix NFTool

**Comment by ketoo** [comment]
I run  NFTool every day, I didn't meet any problems.
If you create new excel files plz follow the format of NF.

**Comment by MiLoserDie** [comment]
What do you use to create excel files?
I use MS Office 2007, but many of my files after creating and running NFTool generate incorrect .ini files, for example, some fields are left blank. I think this is somehow related to the cell format in Excel. Are there any rules when creating Excel files?

In addition, there is another drawback.
This is the order of Property names, they are sorted alphabetically.
In other words, any order created in Excel collapses when using NFTool to generate .ini

**Comment by ketoo** [comment]
The best way to add new excel files is to copy a file that created by NF and change the name as you want.

**Comment by MiLoserDie** [comment]
> The best way to add new excel files is to copy a file that created by NF and change the name as you want.

I do this, but when I need to change the data type from int to string or vice versa, often .ini files are created incorrectly. This is somehow related to the data types in the cells in Excel.

**Comment by ketoo** [comment]
Please show the content as a picture that you have changed.

**Comment by MiLoserDie** [comment]
[ForbiddenName.xlsx](https://github.com/ketoo/NoahGameFrame/files/3293290/ForbiddenName.xlsx)
For example, try use NFTool on this file, he is not correct converted to .ini

**Comment by ketoo** [comment]
I got the result below via tool:

<?xml version='1.0' encoding='utf-8' ?>
<XML>
	<Object Id="Name_0" ="" Name="gm" />
	<Object Id="Name_1" ="" Name="admin" />
	<Object Id="Name_2" ="" Name="support" />
	<Object Id="Name_3" ="" Name="sagaonline" />
</XML>


It looks fine, please check your environment or your permission to run the tool.

**Comment by MiLoserDie** [comment]
> I got the result below via tool:
> 
> ```
> <?xml version='1.0' encoding='utf-8' ?>
> <XML>
> 	<Object Id="Name_0" ="" Name="gm" />
> 	<Object Id="Name_1" ="" Name="admin" />
> 	<Object Id="Name_2" ="" Name="support" />
> 	<Object Id="Name_3" ="" Name="sagaonline" />
> </XML>
> ```
> 
> It looks fine, please check your environment or your permission to run the tool.

No, it's not look fine, see on "<Object Id="Name_0" ="" Name="gm" />", you see = "" ? it's not correct

**Comment by ketoo** [comment]
[ForbiddenName.xlsx](https://github.com/ketoo/NoahGameFrame/files/3293567/ForbiddenName.xlsx)


Please use this file, you have a wrong format as you have a key named "".

**Comment by MiLoserDie** [comment]
> [ForbiddenName.xlsx](https://github.com/ketoo/NoahGameFrame/files/3293567/ForbiddenName.xlsx)
> 
> ```
> <?xml version='1.0' encoding='utf-8' ?>
> <XML>
> 	<Object Id="Name_0" Name="gm" />
> 	<Object Id="Name_1" Name="admin" />
> 	<Object Id="Name_2" Name="support" />
> 	<Object Id="Name_3" Name="sagaonline" />
> </XML>
> ```
> 
> Please use this file, you have a wrong format as you have a key named "".

I have already fixed this file, I just need to figure out how to fix NFTool.
Did you find out why this is happening?
I think this is due to the large number of columns to the right
